metadata/latest/Azure.AI.FormRecognizer.json

```diff
@@ -17,5 +17,8 @@
     "Azure.AI.FormRecognizer.Models",
     "Azure.AI.FormRecognizer.Training",
     "Microsoft.Extensions.Azure"
-  ]
+  ],
+  "DocsConfig": {
+    "Properties": "tfm=netstandard2.0;isPrerelease=true"
```

Comment:
I would suggest making this is a direct property instead of nesting.

Reply:
Sure. Just evaluate the necessities of nested structure. I can't think of any benefits beside of good looking. :)